Cubical/Foundations/Equiv/Dependent.agda

```diff
@@ -166,7 +166,7 @@ fiberIso→IsoOver isom .leftInv  a = isom a .leftInv
 -- Only half-adjoint equivalence can be lifted.
 -- This is another clue that HAE is more natural than isomorphism.
 
-open isHAEquiv
+open isHAEquiv hiding (g)
```

Comment:
I think `using` is clearer than `hiding`. In this case `using (com)` would work.

Reply:
Now I need all of them, so just delete `hiding`I